composer.json

```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": "^7.1",
         "ext-pdo": "*",
-        "doctrine/common": "^2.7.1"
+        "doctrine/common": "^2.7.1",
+        "ocramius/package-versions": "^1.2"
```

Comment:
I know it's already merged into `doctrine/doctrine2` but given the version is displayed in CLI to humans, would it make sense to use `jean85/pretty-package-versions` for prettier output?

Reply:
Eh, this is mostly so we can get proper reports when people get something wrong, so I think the human consumer is mostly @doctrine/doctrinecore

Reply:
Agreed, I would keep it simple and change if requested by users eventually, even now it's 2x more dependencies than before. 😃